### PR TITLE
[SQL] Support hive tables

### PR DESCRIPTION
**Pull request by weiqingy**
Changes:
This is a initial PR to support Hive tables.

Test:
Manually tested in HDP cluster

### CODE REVIEW

**Comment by weiqingy** [comment]
cc: @jerryshao @dongjoon-hyun @bikassaha

File: src/main/scala/com/hortonworks/spark/atlas/types/AtlasEntityUtils.scala [hunk 1]
```diff
@@ -136,27 +138,20 @@ object AtlasEntityUtils extends Logging {
...
   }
 
-  def processToEntity(
-     currUser: String,
-     remoteUser: String,
-     sqlExecutionStart: SparkListenerSQLExecutionStart,
-     sqlExecutionEnd: SparkListenerSQLExecutionEnd,
-     inputs: List[AtlasEntity],
-     outputs: List[AtlasEntity]): AtlasEntity = {
+  def processToEntity(qe: QueryExecution,
```
Comment (dongjoon-hyun) [inline src/main/scala/com/hortonworks/spark/atlas/types/AtlasEntityUtils.scala]: nit. Is this indentation consistent?
Can we reuse Spark style?

**Comment by dongjoon-hyun** [comment]
@weiqingy . I will merge this to proceed since we are very initial stage of development.

**Comment by dongjoon-hyun** [comment]
Oops. I should do squash and merge. :(

File: src/main/scala/com/hortonworks/spark/atlas/types/AtlasEntityUtils.scala (hunk 1 above)
Comment (weiqingy) [reply src/main/scala/com/hortonworks/spark/atlas/types/AtlasEntityUtils.scala]: Yes. Sure. I'll change the coding style.

**Comment by weiqingy** [comment]
Thanks, @dongjoon-hyun . I'll submit another PR very soon.

**Pull request merged**